src/HtmlUtils.tsx

```diff
@@ -601,6 +613,48 @@ export function bodyToHtml(content: IContent, highlights: string[], opts: IOpts
         </span>;
 }
 
+/* turn a room topic into html
+ *
+ * topic: plain text topic
+ * htmlTopic: optional html topic
+ * ref: React ref to attach to any React components returned
+ */
+export function topicToHtml(topic: string, htmlTopic?: string, ref?: React.Ref<HTMLSpanElement>): ReactNode {
+    if (!SettingsStore.getValue("feature_html_topic")) {
+        htmlTopic = null;
+    }
+
+    const isFormattedTopic = !!htmlTopic;
+    let topicHasEmoji = false;
+    let safeTopic = "";
+
+    try {
+        topicHasEmoji = mightContainEmoji(isFormattedTopic ? htmlTopic : topic);
+
+        if (isFormattedTopic) {
+            safeTopic = sanitizeHtml(htmlTopic, topicSanitizeHtmlParams);
+            if (topicHasEmoji) {
+                safeTopic = formatEmojis(safeTopic, true).join('');
+            }
+        }
+    } catch {}
```

Comment:
anytime we ignore a catch we should have a comment for why. Eg: `// ignore because worst case is we assume it's not HTML`

Reply:
... though we should probably set `isFormattedTopic = false` explicitly so we render *a* topic.

Reply:
Good catch. This was left empty after removing the unrelated code copied from `bodyToHtml`. I set the is-formatted flag back to `false` in the `catch` now.